app/src/main/java/com/nexenio/bleindoorpositioningdemo/ui/beaconview/map/BeaconMap.java

```diff
@@ -41,6 +43,7 @@
     protected LocationAnimator bottomRightLocationAnimator;
 
     protected CanvasProjection canvasProjection;
+    protected DeviceLocationPredictor deviceLocationPredictor;
```

Comment:
Location prediction should be part of the `IndoorPositioning` singleton, not some random view that is not part of the library.